VimeoNetworking/Sources/Models/Subscription.h

```diff
@@ -0,0 +1,13 @@
+//
+//  Subscription.h
+//  Pods
+//
+//  Created by Lim, Jennifer on 2/8/17.
+//
+//
+
+#import <Foundation/Foundation.h>
```

Comment:
```Objc
@import Foundation
```
Are these models going to be filled out in a later PR, and this is just a placeholder?